stable/cockroachdb/README.md

```diff
@@ -227,80 +227,90 @@ Verify that no pod is deleted and then upgrade as normal. A new StatefulSet will
+| `statefulset.secretMounts`                       | Additional Secrets to mount at cluster members                  | `[]`                                             |
+| `statefulset.labels`                             | Additional labels of StatefulSet and its Pods                   | `{"app.kubernetes.io/component": "cockroachdb"}` |
+| `statefulset.annotations`                        | Additional annotations of StatefulSet Pods                      | `{}`                                             |
+| `statefulset.nodeAffinity`                       | [Node affinity rules][2] of StatefulSet Pods                    | `{}`                                             |
+| `statefulset.podAffinity`                        | [Inter-Pod affinity rules][1] of StatefulSet Pods               | `{}`                                             |
+| `statefulset.podAntiAffinity`                    | [Anti-affinity rules][1] of StatefulSet Pods                    | auto                                             |
+| `statefulset.podAntiAffinity.type`               | Type of auto [anti-affinity rules][1]                           | `soft`                                           |
+| `statefulset.podAntiAffinity.weight`             | Weight for `soft` auto [anti-affinity rules][1]                 | `100`                                            |
+| `statefulset.nodeSelector`                       | Node labels for StatefulSet Pods assignment                     | `{}`                                             |
+| `statefulset.tolerations`                        | Node taints to tolerate by StatefulSet Pods                     | `[]`                                             |
+| `statefulset.resources`                          | Resource requests and limits for StatefulSet Pods               | `{}`                                             |
+| `statefulset.livenessProbe.enabled`              | Whether to use a liveness probe on the StatefulSet pods         | `no`                                             |
+| `statefulset.livenessProbe.initialDelaySeconds`  | Delay after pod starts before liveness probes start             | `30`                                             |
+| `statefulset.livenessProbe.periodSeconds`        | Frequency to send liveness probes to StatefulSet Pods           | `5`                                              |
+| `statefulset.livenessProbe.timeoutSeconds`       | Timeout on liveness probes                                      | `1`                                              |
+| `statefulset.livenessProbe.failureThreshold`     | Consecutive liveness probe failures before container restarts   | `3`                                              |
+| `statefulset.readinesssProbe.enabled`            | Whether to use a liveness probe on the StatefulSet pods         | `yes`                                            |
```

Comment:
Description here and below mention liveness probe but should mention readiness.

Reply:
Fixed.